pom.xml

```diff
@@ -93,6 +97,27 @@
             <artifactId>allure-junit5</artifactId>
             <version>2.19.0</version>
         </dependency>
+        <dependency>
+            <groupId>net.joshka</groupId>
+            <artifactId>junit-json-params</artifactId>
+            <version>5.6.2-r0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.json</artifactId>
+            <version>1.1.4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-firefox-driver</artifactId>
```

Comment:
you can use 
```
<dependency>
            <groupId>org.seleniumhq.selenium</groupId>
            <artifactId>selenium-java</artifactId>
            <version>4.4.0</version>
        </dependency>
```
instead of `selenium-support`, `selenium-firefox-driver`, `selenium-chrome-driver`

Reply:
fixed